lib/shopify_api/context.rb

```diff
@@ -131,7 +134,7 @@ def embedded?
 
       sig { returns(T::Boolean) }
       def setup?
-        !(api_key.empty? || api_secret_key.empty? || host_name.empty?)
+        !(api_key.empty? || api_secret_key.empty? || host_name.empty? || T.must(host).empty?)
```

Comment:
Just checking that `host` will always be present here - even if only `host_name` is passed in?

Reply:
That's right, in the context we'll assign the default for `host` like so:

```ruby
host: ENV["HOST"] || "https://#{host_name}"
```